packages/shared/features/onboardingFeatures.ts

```diff
@@ -134,7 +134,7 @@ const onboardingFeatures = {
         devnet: {
             enabled: true,
             claimRewards: {
-                enabled: false,
+                enabled: true,
```

Comment:
should not be enabled yet because the there isn't a single flow complete

Reply:
We should also add feature flags for the three different paths

Reply:
> We should also add feature flags for the three different paths

I agree, although let's make this change in another PR where we can also properly type the `onboardingFeatures` object :+1:

Reply:
I disagree properly typing is a larger task. You added a partial flow for shimmer claiming on software profiles but not ledger profiles. So this should be feature flagged